### PR TITLE
4.x: Add config for zero-token nodes & ZeroTokenNodesIT

### PR DESCRIPTION
**Pull request by Bouncheck**
Add ZeroTokenNodesIT:

Adds a test class verifying the behavior of the driver when dealing with
zero-token Scylla nodes.

Adds a configurable option which controls whether to allow zero-token nodes as
valid peers. Until now such nodes were considered invalid. Default value of
this option does not change the driver behavior.

Bundled with CcmBridge adjustments:

Adds `startWithArgs` methods that allow avoiding the default arguments like
"--wait-other-notice". This wait causes timeout in multi-dc test
utilizing `join_ring: false` so it needs to be avoided.

Adds `addWithoutStart(n, dc)`. As the name suggests it's an add that does not
immediately start the node.

Adds `updateNodeConfig` in order to allow adding new nodes with different
configurations.

### CODE REVIEW

File: test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java [hunk 1]
```diff
@@ -373,7 +373,7 @@ public void reloadCore(int node, String keyspace, String table, boolean reindex)
   public void start() {
     if (started.compareAndSet(false, true)) {
       try {
-        execute("start", jvmArgs, "--wait-for-binary-proto", "--wait-other-notice");
+        execute("start", jvmArgs, "--wait-for-binary-proto");
```
Comment (dkropachev) [inline test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java]: Why did you remove it ?
It will change CCM behavior for all tests and can cause tests instability.

**Review by dkropachev — CHANGES_REQUESTED** [review]

File: test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java (hunk 1 above)
Comment (Bouncheck) [reply test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java]: It clashes with `join_ring` set to false. It will timeout. IIRC we added this on a whim (i don't think it was fixing any test in particular, we just thought it's better to have it than not) not that long ago, so I just removed it here. Other option would be making this configurable, so that it can be disabled just for zero token tests.

Comment (dkropachev) [reply test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java]: Let's add another public method, startNoWait and use it for `join_ring=false` nodes

Comment (Bouncheck) [reply test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java]: Added `startWithArgs` that uses custom arguments. Old `start` should be functionally the same as before.

**Review by dkropachev — APPROVED** [review]
